draft-ietf-rats-architecture.md

```diff
@@ -89,12 +89,14 @@ In Remote Attestation Procedures (RATS), one peer (the "Attester") produces beli
 RATS are facilitated by an additional vital party, the Verifier.
 The Verifier appraises Evidence via Appraisal Policies and creates the Attestation Results to support Relying Parties in their decision process.
 
-This documents defines a flexible architecture with corresponding roles and their interaction via conceptual messages.
+This documents defines a flexible architecture consisting of attestation Roles and their interactions via Role Messages.
```

Comment:
Change "Role Messages" back to "conceptual messages".  The latter is the name of the section title, and we don't need to define a new term.
```suggestion
This documents defines a flexible architecture consisting of attestation Roles and their interactions via conceptual messages.
```